python_transport/wirepas_gateway/protocol/mqtt_wrapper.py

```diff
@@ -40,8 +38,13 @@ def __init__(
         self.logger = logger
         self.on_termination_cb = on_termination_cb
         self.on_connect_cb = on_connect_cb
+        # Set to track the unpublished packets
+        self._unpublished_mid_set = set()
+        # Variable to keep track of latest published packet
+        self._timestamp_last_publish = datetime.now()
```

Comment:
Should the timestamp be time zone aware? You can set one, eg, utc by setting the tz argument https://docs.python.org/3.7/library/datetime.html#datetime.datetime.now

I've run into issues in the past where a UTC timestamp was interpret as local time, since the +00 was left out. But that was due to the datetime -> isoformat -> datetime. 

If this is left as unix timestamp, then it is okay as is.

Reply:
I don't think it makes any difference. It is mainly internal use and not exported. As long as all timestamp are taken the same way, it shouldn't matter as we just care about delta

Reply:
Agree. Safe to leave it as is.